### PR TITLE
Add filtering negative feedback in the test set

### PR DESCRIPTION
**Pull request by ita9naiwa**
In train_test_split function (in evaluation.pyx),
There is no filtering negative feedback there. This might make some (1) wrong predictions hit or (2) result lower recall score because of regarding uninteresting items as interesting ones.

### CODE REVIEW

**Comment by benfred** [comment]
thanks!
